### DISCUSSION TITLE
Where is data saved?

### DISCUSSION THREAD

**Discussion by marie-dk** [cyrilletuzi/angular-async-local-storage#43] [Q&A] [answered]
I've been banging my head against the wall trying to figure out why data was not saved... But then I tried fetching data, and its there! It's just not visible in Chromes inspector. I opened Firefox, but it's not there either.

Where is data saved? What am I missing? 

Thanks :-)

**Comment by darthmolen** [comment, marked as answer]
There's an "IndexedDB" tab in chrome developer and you'll find your data there. Although this module is named "LocalStorage", it is just an emulation for that synchronous product. So in reality, it stores it in IndexedDB.

**Comment by marie-dk** [comment]
Oh God... I feel so stupid. I did check indexedDB but missed the extra level I needed to go down to actually see the data, LOL! Thanks :-)
